### PR TITLE
SG-21557 other integrations

### PR DESCRIPTION
**Pull request by zamu5**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/92002783074c7d16a200e5afe6bde17d797033ef/index.html)

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:14]:
```suggestion
Cinesync allows you to have simultaneous synced playback between multiple locations. {% include product %}'s integration allows you to create a Playlist of Versions, play it in Cinesync, and send your Notes made during the session right back into {% include product %}.

For more information, please see [http://www.cinesync.com/manual/latest](http://www.cinesync.com/manual/latest/).
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:18]:
```suggestion
The {% include product %}+Deadline integration allows you to automatically submit rendered Versions to {% include product %} complete with thumbnail, links to frames, and other metadata.

For more information, please see [http://www.thinkboxsoftware.com/deadline-5-shotgunevent](http://www.thinkboxsoftware.com/deadline-5-shotgunevent).
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:22]:
```suggestion
Much like the Deadline integration, the {% include product %}+Rush integration allows you to automatically submit rendered Versions to {% include product %} complete with thumbnail, links to frames, and other metadata.

For more information, please see [http://seriss.com/rush-current/index.html](http://seriss.com/rush-current/index.html).
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:26]:
```suggestion
{% include product %} a light but flexible integration, which we use internally, that allows us to track revisions and link them to tickets and releases in {% include product %}. We also provide links to Trac to integrate with an external web SVN repository viewer. This is all done by adding a post-commit hook to SVN, a {% include product %} API script that takes some ENV variables from the commit and then creates a Revision entity in {% include product %} with various fields filled in. It can be modified to match your studio's needs and can be used for a local or hosted installation since it's just using the API.

For more information, please see [https://subversion.apache.org/docs](https://subversion.apache.org/docs/).
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:12]:
```suggestion
## Cinesync
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:16]:
```suggestion
## Deadline
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:20]:
```suggestion
## Rush
```

File: docs/en/guides/pipeline-integrations/administration/other-integrations.md
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/other-integrations.md:24]:
```suggestion
## Subversion (SVN)
```

**Review by thebeeland — CHANGES_REQUESTED** [review]
A few minor formatting issues. It should be good to go once those are added, though I'll have one last look at a preview build before merging once they've been added in.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c3658305cb86b0cd4263cc8f382eeda9d8378850/index.html)

**Review by thebeeland — APPROVED** [review]
Looks great! I'm going to merge it now.

**Pull request merged**
